src/main/kotlin/sg/com/quantai/etl/controllers/CryptoController.kt

```diff
@@ -0,0 +1,48 @@
+package sg.com.quantai.etl.controllers
+
+import com.fasterxml.jackson.databind.JsonNode
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import sg.com.quantai.etl.services.CryptoService
+
+@RestController
+@RequestMapping("/crypto")
+class CryptoController(private val cryptoService: CryptoService) {
+
+    /**
+     * Fetch the current price of a cryptocurrency
+     * Example Usage:
+     * GET /crypto/current/price?symbol=BTC&currency=USD
+     */
+    @GetMapping("/price/current")
+    fun getCryptoPrice(
```

Comment:
This functionality/endpoint is not connected to any User Story (182, 184 and 254). It should be removed, then.

Reply:
Yep , I created this endpoint just to test out one of the API endpoint at the earlier stages of development

Reply:
Removed